nucliadb/nucliadb/ingest/orm/processor/__init__.py

```diff
@@ -271,6 +272,13 @@ async def txn(
                     partition=partition, seqid=seqid, multi=multi, kbid=kbid, rid=uuid
                 )
                 logger.warning(f"This message did not modify the resource")
+        except (asyncio.TimeoutError, asyncio.CancelledError):  # pragma: no cover
```

Comment:
Should we catch ConnectionError too? (both server and client errors)

Reply:
Yes, agreed. Let me see if there is a better way to do this.

Reply:
Maybe all those exceptions that we define as retriable should be catched so we hard-fail (and retry even more on the next processing attempt)